### DISCUSSION TITLE
Add .hdf suffix

### DISCUSSION THREAD

**Discussion by Aeroid** [ZuluSCSI/ZuluSCSI-firmware#151] [Ideas]
I know it is just small inconvenience, but if ".hdf" would be supported as an alternative suffix along side the ".hda" and ".img" suffixes, it would be quite easy to use the SD Card files directly within AmigaForever/winUAE w/o renaming them back and forth when swapping the card between a physical and an emulated Amiga.

**Comment by aperezbios** [comment]
@Aeroid it already is supported. ZuluSCSI doesn't care what the file name suffix is, with the exception of a blacklist of compressed file suffixes, such as .zip, rar, .gz, etc. 

You could take a file and name it HD4.xyz and it would work.

**Reply by Aeroid** [reply]
Thanks, I guess I should have more trust in such a great tool :-)
